### ISSUE TITLE
Failed to run first app, SyntaxError: Unexpected token '.'

### ISSUE THREAD

**Issue by ikrz** [neutralinojs/neutralinojs-cli#97]
I have followed Your First N App instructions, an can not run neu.
My OS: Ubuntu 21.10
I did:
$ sudo npm install -g @neutralinojs/neu
added 143 packages, and audited 144 packages in 7s

12 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Then run:
$ neu create test1
/usr/local/lib/node_modules/@neutralinojs/neu/src/modules/filewatcher.js:30
    if(configObj?.cli?.autoReloadExclude) {
                 ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@neutralinojs/neu/src/commands/run.js:3:21)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@neutralinojs/neu/src/neu-cli.js:2:13)
    at Module._compile (internal/modules/cjs/loader.js:999:30)

Same happens if I run neu version.

Then I tried npx: 
$ npx @neutralinojs/neu version
Need to install the following packages:
  @neutralinojs/neu
Ok to proceed? (y) 
/home/ikriznar/.npm/_npx/0214858268217ccb/node_modules/@neutralinojs/neu/src/modules/filewatcher.js:30
    if(configObj?.cli?.autoReloadExclude) {
                 ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/ikriznar/.npm/_npx/0214858268217ccb/node_modules/@neutralinojs/neu/src/commands/run.js:3:21)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/ikriznar/.npm/_npx/0214858268217ccb/node_modules/@neutralinojs/neu/src/neu-cli.js:2:13)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
npm ERR! code 1
npm ERR! path /home/ikriznar/webapps
npm ERR! command failed
npm ERR! command sh -c neu "version"

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ikr/.npm/_logs/2022-01-12T20_13_53_853Z-debug.log

**Comment by wmaurer** [comment]
I encountered the same problem and have managed to resolve it by using Node v14.
The code is failing because of the optional chaining operator (?.), which is not supported any version of node less than 14.

**Comment by ikrz** [comment]
Thx for the tip, I have found solution at the end. I have naively expect that whole point of npm is to poll right version for installation, but it is more complicated. After some digging I have found out, that out-of-the-box in Ubuntu I get versions:
npm: '7.5.2',
node: '12.22.5'

I did following:
npm install -g npm
npm install -g node
And this elevated my effective versions to:
npm: '8.3.1',
node: '17.3.1'

This fixed the problem. Thx.
